src/libs/CardUtils.ts

```diff
@@ -72,11 +73,13 @@ function getYearFromExpirationDateString(expirationDateString: string) {
  * @param cardList - collection of assigned cards
  * @returns collection of assigned cards grouped by domain
  */
-function getDomainCards(cardList: Record<string, OnyxTypes.Card>) {
+function getDomainCards(cardList: OnyxCollection<OnyxTypes.Card>) {
     // Check for domainName to filter out personal credit cards.
-    // eslint-disable-next-line you-dont-need-lodash-underscore/filter
-    const activeCards = lodash.filter(cardList, (card) => !!card.domainName && (CONST.EXPENSIFY_CARD.ACTIVE_STATES as ReadonlyArray<OnyxTypes.Card['state']>).includes(card.state));
-    return lodash.groupBy(activeCards, (card) => card.domainName);
+    const activeCards = Object.values(cardList ?? {}).filter(
+        (card) => !!card?.domainName && (CONST.EXPENSIFY_CARD.ACTIVE_STATES as ReadonlyArray<OnyxTypes.Card['state']>).includes(card.state),
```

Comment:
You can use `.some` instead of `.includes` to get rid of assertion

Reply:
updated